python/mxnet/executor.py

```diff
@@ -311,212 +309,3 @@ def debug_str(self):
             self.handle, ctypes.byref(debug_str)))
         return py_str(debug_str.value)
 
-def _split_input_slice(batch_size, work_load_list):
```

Comment:
I think this code was originally in model.py, later moved to executor.py due to dataparallelexecutormanager. I moved it to executor_manager.py here.